static/app/components/searchQueryBuilder/tokens/filterKeyListBox/index.tsx

```diff
@@ -212,7 +207,7 @@ function FilterKeyMenuContent<T extends SelectOptionOrSectionWithKey<string>>({
   sections,
 }: FilterKeyMenuContentProps<T>) {
   const {filterKeys} = useSearchQueryBuilder();
-  const focusedItem = state.collection.getItem(state.selectionManager.focusedKey)?.props
+  const focusedItem = state.collection.getItem(state.selectionManager.focusedKey!)?.props
```

Comment:
in the other PR I opened, I had fixed this differently without a no-null assertion:

```
  const focusedItem: string | undefined = state.selectionManager.focusedKey
    ? state.collection.getItem(state.selectionManager.focusedKey)?.props?.value
    : undefined;
```

Reply:
this isn’t resolved yet, we’re still using the no-null assertion here even though this is fixable with a runtime check